src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -67,6 +67,12 @@
     protected FcrepoClient.FcrepoClientBuilder clientBuilder;
 
     /**
+     * The file, within the export directory for the directory mirrors the container
```

Comment:
Yeah, I'll try again, though this variable will mostly be useful in the future when we can export from one repo/path and import to another.